source/usage-examples/command.txt

```diff
@@ -5,23 +5,33 @@ Run a Command
 .. default-domain:: mongodb
 
 You can run all raw database operations, not including
-:manual:`CRUD </crud/>` operations, using the :node-api:`command()
-<Admin.html#~command>` method. Call the ``command()`` method with
-your command object on an instance of ``Admin`` for diagnostic and
+:manual:`CRUD </crud/>` operations, using the :node-api:`db.command()
```

Comment:
I think this guidance is incorrect. You can run _all_ database operations_including_ CRUD commands through the `Db#command` helper. 

Additionally, I think we can probably just outright remove this section. Originally, we provided a helper class called `Admin` to provide builders for administrative tasks. We want to remove that in v4 of the driver, and have users depend instead on using `command` for admin tasks, or really just use the shell.

Reply:
We chose to include this method since it was one of the most-searched-for methods. If I remove mention of not supporting CRUD operations and address the other comments, would you be ok with keeping it?